### PR TITLE
PR: Add Spyder 2022 roadmap and proposals blogpost

### PR DESCRIPTION
**Pull request by dalthviz**

### CODE REVIEW

**Comment by CAM-Gerlach** [comment]
I'll review this for content and meanwhile upgrade the pinned deps (including the Lektor version) and linting checks in another PR, which should fix the one unrelated linter issue.

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 1]
```diff
@@ -0,0 +1,43 @@
+title: Spyder 2022 roadmap and new grants work
+---
+author: dalthviz
+---
+category: News
+---
+tags: milestone, team, status-update
+---
+pub_date: 2022-07-18 20:00
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]: Just as a reminder to whomever merges: Don't forget to update this!

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+---
+summary: A quick update on Spyder 2022 roadmap and proposals being worked on.
+---
+body:
+
+For the last couple of months the Spyder team has been working on defining a roadmap and submitting grant proposals.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
For the last couple of months, the Spyder team has been working on defining a new roadmap and submitting grant proposals to fund more features and improvements.
```

* Add compelling motivation for the grant proposals
* Avoid implying that we didn't have _any_ roadmap before
* Add missing comma

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Now we are pleased to announce that we have a roadmap and we got two proposals accepted!
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
We are pleased to announce our roadmap for the rest of 2022, and that two proposals were funded!
```

* Clarify details and make clear that we're announcing the roadmap itself, not just the mere fact that we have one
* Fix grammar and phrasing issues
* Use more appropriate/specific diction on proposal acceptance
* Elide unneeded "Now"

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## The roadmap
+
+Thinking on the importance of sharing a clear perspective of where the Spyder project is going and where most of our efforts will go for the next months, the Spyder team have created [an initial roadmap for the rest of 2022](https://github.com/spyder-ide/spyder/wiki/Roadmap).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
Considering the importance of sharing a clear perspective of where the Spyder project is going and where we will be focusing our efforts over the coming months, the team has created [an initial roadmap for the rest of 2022](https://github.com/spyder-ide/spyder/wiki/Roadmap).
```

* Use grammatically and semantically correct wording ("Thinking on" -> "Considering")
* Fix grammar (plural/singular agreement & "next [sic] months")
* Use more appropriate and idiomatic phrasing to describe focusing our efforts
* Avoid repetition of "Spyder"

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+The elements included in the roadmap were selected since we think those are important features and characteristics that have been mentioned in issues, face-to-face or virtual discussions, Stack Overflow, social media and in general feedback that we have received and that we think encapsulate our users and community interests.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
We prioritized the highlighted features and enhancements based on input from issues, face-to-face and virtual discussions, Stack Overflow, social media and other feedback, to try to best capture the interests of our users and community.
```

* Rewrote to make much more focused and concise, and avoid a run-on sentence
* Reword to be more compelling
* Avoid passive voice and awkward phrasing

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+## The proposals
+
+As part of making our roadmap achievable we took time to write and present proposals to different venues and organizations in the last couple of months.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
To help make our roadmap achievable, we wrote and submitted proposals to several different venues and organizations in the last couple of months.
```

* Use simpler, shorter and more direct phrasing
* Use accurate diction ("present" -> "submit")
* Add missing comma

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+We have yet to hear back for some proposals to be granted but as mentioned we got two accepted!
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
While we have yet to hear back from some of them, two have already been funded!
```

* Fix several significant grammar issues
* Rewrite to be more focused, clear and concise
* Improve phrasing to be easier to understand

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+The first one we got was the [Google Season of Docs 2022 (GSoD 2022)](https://developers.google.com/season-of-docs/docs).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
The first was for the [Google Season of Docs (GSoD) 2022)](https://developers.google.com/season-of-docs/docs).
```

* Simplify awkward, verbose and unnecessary phrasing 
* Avoid duplicate 2022

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+This grant will be focused mainly on improving and updating our users documentation for Spyder 5 Editor features.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
This grant will be focused mainly on improving and updating our user documentation for Spyder 5's Editor features.
```

* Fix two small grammar/syntax errors

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+It also will help move forward our developers' documentation and make available tutorials and detailed information about our new plugin API.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
It also will help enhance our developer documentation and add tutorials and detailed information covering our new plugin API.
```

* Fix grammar and wording issues

I'm not sure how much of this is still part of the current plan. @steff456 , can you confirm?

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+For more details about this grant you can check the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/GSoD-2022-Project-Proposal).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
For more details, check out the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/GSoD-2022-Project-Proposal).
```

* Simplify awkward and wordy phrasing
* Fix grammar issue
* Add missing comma

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+The second proposal accepted was a [Small Development Grant from NumFOCUS](https://numfocus.org/programs/small-development-grants).
+In this grant we will be working on improving the installation experience for Spyder Windows users, by enabling the current standalone installers available to be auto-updatable and providing a way to install custom packages that are not bundled with the installers in a simpler way.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
For this grant, we will be working on improving the installation experience for Windows users by making the standalone installers auto-updatable and providing a simpler way to install custom packages not bundled with the installers.
```

* Fix grammar/prepositions
* Tweak phrasing to be shorter, simpler and easier to understand
* Use more appropriate wording
* Fix comma usage

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+With this grant we were able to contract one new developer, Juan Sebastian Bautista, a recent Systems and Computing engineer graduate from Universidad de los Andes in Bogotá, Colombia. Juan Sebastian will help us implement the proposal working alongside with the Spyder team in a six months time frame.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
With this grant, we were able to hire a new developer—Juan Sebastian Bautista, a recent graduate in systems and computing engineering from Universidad de los Andes in Bogotá, Colombia—who will implement these features over the next six months.
```

* Fix grammar and clarify phrasing around degree/graduation
* Set off introduction of Juan Sebastian into a parenthetical to improve flow
* Simplify redundant/verbose phrasing and be clearer and more concise
* Use more appropriate wording
* Add missing comma
* Fix capitalization of academic major

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+For more details about this grant you can check the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/NumFOCUS-Small-Development-Grant-2022).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
For more details, see the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/NumFOCUS-Small-Development-Grant-2022).
```

* Greatly simplify redundant phrasing
* Fix grammar
* Vary the words a bit from the previous

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+
+## Next steps
+
+We will be focusing our time as a project on pushing forward the accepted proposals and submitting new ones.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
We will be focusing our efforts as a project on moving forward with the accepted proposals and submitting new ones.
```

* Fix grammar
* Use more appropriate words

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+On top of this we will try to get more feedback about the things that volunteers, users and community in general want to see in the future for Spyder and share the proposals progress relaunching the [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
On top of that, we will gather more feedback about what volunteers, users and the community at large want to see for Spyder in the future, and share the progress of the proposals in the relaunched [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```

* Rewrite to fix unclear and difficult to understand grammar and meaning in the last clause
* Fix various grammar and wording issues
* Use more appropriate diction
* Add missing comma

I'm not 100% sure what you intended to say at the end, but I tried to infer what I could of its meaning here.

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+
+As a side note we want to thank NumFOCUS and Google Season of Docs for granting us the funds to be able to work on these projects and start making our roadmap possible!
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
We would like to thank NumFOCUS and Google Season of Docs for granting us the funds to be able to work on these projects and making our roadmap possible!
```

* Use simpler and more appropriate phrasing
* Avoid missing comma

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 18]
```diff
@@ -0,0 +1,43 @@
+title: Spyder 2022 roadmap and new grants work
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
title: New 2022 roadmap and grant funding
```

* Emphasize that both are new
* Elide redundant "Spyder" (this is the Spyder blog)
* Use more appropriate and compelling word ("work" -> "funding")

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+---
+author: dalthviz
+---
+category: News
+---
+tags: milestone, team, status-update
+---
+pub_date: 2022-07-18 20:00
+---
+summary: A quick update on Spyder 2022 roadmap and proposals being worked on.
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
summary: We'd like to share our brand new roadmap for the rest of 2022, powered by the feedback of users like you. We're also excited to announce two new grants we've been awarded, which fund significant improvements to Spyder's Windows installers and user/developer documentation.
```

I've rewritten this to roughly match the length, tone and content of the other summaries, with a short paragraph describing the key points, while sounding compelling and enticing viewers to click through and read more.

**Review by CAM-Gerlach — CHANGES_REQUESTED** [review]
Thanks, @dalthviz , looks good overall. I just have some comments, all as suggestions, to improve the title and summary, and the grammar, phrasing, wording and punctuation of the body text.

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+
+The second proposal accepted was a [Small Development Grant from NumFOCUS](https://numfocus.org/programs/small-development-grants).
+For this grant, we will be working on improving the installation experience for Windows users by making the standalone installers auto-updatable and providing a simpler way to install custom packages not bundled with the installers.
+With this grant, we were able to hire a new developer—Juan Sebastian Bautista, a recent graduate in systems and computing engineering from Universidad de los Andes in Bogotá, Colombia—who will implement these features over the next six months.
+For more details, see the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/NumFOCUS-Small-Development-Grant-2022).
+
+## Next steps
+
+We will be focusing our efforts as a project on moving forward with the accepted proposals and submitting new ones.
+On top of that, we will gather more feedback about what volunteers, users and the community at large want to see for Spyder in the future, and share the progress of the proposals in the relaunched [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```
Comment (ccordoba12) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
On top of that, we will gather more feedback about what volunteers, users and the community at large want to see for Spyder in the future version 6, and share the progress of the proposals in the relaunched [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```

**Review by ccordoba12 — COMMENTED** [review]
This is a pretty nice blogpost, thanks @dalthviz!

I left a tiny suggestion for you, otherwise looks good to me.

File: content/blog/spyder-roadmap-proposals-2022/contents.lr [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+
+The second proposal accepted was a [Small Development Grant from NumFOCUS](https://numfocus.org/programs/small-development-grants).
+For this grant, we will be working on improving the installation experience for Windows users by making the standalone installers auto-updatable and providing a simpler way to install custom packages not bundled with the installers.
+With this grant, we were able to hire a new developer—Juan Sebastian Bautista, a recent graduate in systems and computing engineering from Universidad de los Andes in Bogotá, Colombia—who will implement these features over the next six months.
+For more details, see the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/NumFOCUS-Small-Development-Grant-2022).
+
+## Next steps
+
+We will be focusing our efforts as a project on moving forward with the accepted proposals and submitting new ones.
+On top of that, we will gather more feedback about what volunteers, users and the community at large want to see for Spyder in the future version 6, and share the progress of the proposals in the relaunched [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```
Comment (CAM-Gerlach) [inline content/blog/spyder-roadmap-proposals-2022/contents.lr]:
```suggestion
On top of that, we will gather more feedback about what volunteers, users and the community at large want to see for Spyder 6, and share the progress of the proposals in the relaunched [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```

Shorter, simpler and easier to read

**Comment by CAM-Gerlach** [comment]
#207 updates the build system and pre-commit hooks, resolving the one failing check, and should be ready to merge. Then, you can rebase this on that and everything should be :heavy_check_mark: (tested and confirmed locally for builds and pre-commit).

**Comment by CAM-Gerlach** [comment]
#207 is now merged, so this can be rebased and it should be good to go

**Review by CAM-Gerlach — DISMISSED** [review]
LGTM, thanks @dalthviz . Just don't forget to update the data before merging.

**Comment by dalthviz** [comment]
Done @CAM-Gerlach :+1:

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz! Really nice blogpost.

**Comment by ccordoba12** [comment]
I'm going to merge this one because @CAM-Gerlach already approved it.

**Pull request merged**